### PR TITLE
Add recipe for AvatarizeBundle

### PR DESCRIPTION
**Pull request by birkof**
| Q             | A
| ------------- | ---
| License       | MIT
| Version       | 0.1.3

### CODE REVIEW

File: birkof/avatarize-bundle/0.1/config/packages/avatarize.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+ass:
```
[the remark this answers is not in the document]
Comment (stof) [reply birkof/avatarize-bundle/0.1/config/packages/avatarize.yaml]: thus, this is broken as it is not even your DI alias

**Comment by stof** [comment]
The doc is quite broken too: https://github.com/birkof/avatarize-bundle/blob/6ed2cfad2afd52365a4de38549c41d388551e0ca/src/Resources/doc/index.md

**Comment by birkof** [comment]
Damn, that shouldn't be there ... Monitor brightness tricked me :D

**Comment by birkof** [comment]
Also, the documentation is in progress...

**Comment by Nyholm** [comment]
Flex is smart enough to install this bundle without a recipe. Im going to close this PR.

**Pull request closed without merging**